platform/features/timeline/bundle.js

```diff
@@ -182,6 +182,16 @@ define([
                                 "capacity"
                             ],
                             "pattern": "^-?\\d+(\\.\\d*)?$"
+                        },
+                        {
+                            "name": "Battery Starting SOC",
+                            "control": "textfield",
+                            "required": false,
+                            "conversion": "number",
+                            "property": [
+                                "startingSOC"
+                            ],
+                            "pattern": "^-?\\d+(\\.\\d*)?$"
```

Comment:
It's now updated to percentage. Haven't started on the `startingSOC > capacity` issue yet but would be glad to dig into it.